app/controllers/api/v3/facilities_controller.rb

```diff
@@ -16,9 +16,11 @@ def current_facility_records
   end
 
   def other_facility_records
-    Facility
-      .with_discarded
-      .updated_on_server_since(other_facilities_processed_since, limit)
+    Statsd.instance.time("other_facility_records.Facility") do
```

Comment:
These `time` calls are now timing the old sync code (post revert), but whenever we ship the new-new perf changes, the body can be swapped and the timer can stay.